### PR TITLE
Fix rt tf

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_evaluators/evaluators/realtime_evaluator.py [hunk 1]
```diff
@@ -53,6 +54,8 @@ def _get_tentacle_registration_topic(self, all_symbols_by_crypto_currencies, tim
             if ideal_time_frame != to_handle_time_frame:
                 self.logger.warning(f"Missing {ideal_time_frame.name} time frame in available time frames, "
                                     f"using {to_handle_time_frame.name} instead.")
+            # set self.available_time_frame with the actually available one
+            self.available_time_frame = to_handle_time_frame.value
```
Comment (Herklos) [inline octobot_evaluators/evaluators/realtime_evaluator.py:58]: Why not setting `self.time_frame`?

File: octobot_evaluators/util/evaluation_util.py [hunk 2]
```diff
@@ -33,6 +34,6 @@ def get_shortest_time_frame(ideal_time_frame, preferred_available_time_frames, o
     if ideal_time_frame in preferred_available_time_frames:
         return ideal_time_frame
     if preferred_available_time_frames:
-        return preferred_available_time_frames[0]
+        return time_frame_manager.sort_time_frames(preferred_available_time_frames)[0]
     else:
-        return others[0]
+        return time_frame_manager.sort_time_frames(others)[0]
```
Comment (Herklos) [inline octobot_evaluators/util/evaluation_util.py:39]: :+1:

File: octobot_evaluators/evaluators/realtime_evaluator.py (hunk 1 above)
Comment (GuillaumeDSM) [reply octobot_evaluators/evaluators/realtime_evaluator.py:58]: I did it at first but `self.time_frame` is read from config, I think it's better not to erase it for clarity

Comment (Herklos) [reply octobot_evaluators/evaluators/realtime_evaluator.py:58]: ok :+1:

**Review by Herklos — APPROVED** [review]

**Pull request merged**